DiffAugment-stylegan2-pytorch/training/training_loop.py

```diff
@@ -369,17 +376,12 @@ def training_loop(
         # Log image and text to Comet.ml
         if rank == 0 and comet_api_key and comet_ml is not None:
             try:
-                experiment = comet_ml.ExistingExperiment(api_key=comet_api_key,
-                                                         previous_experiment=comet_experiment_key,
-                                                         auto_output_logging=False, auto_log_co2=False,
-                                                         auto_metric_logging=False, auto_param_logging=False,
-                                                         display_summary_level=0)
-                if (image_snapshot_ticks is not None) and (done or cur_tick % image_snapshot_ticks == 0):
+                if experiment is not None and (image_snapshot_ticks is not None) and (done or cur_tick % image_snapshot_ticks == 0):
                     experiment.log_image(image_data=os.path.join(run_dir, f'fakes{cur_nimg//1000:06d}.png'),
                                          name=f'fakes{cur_nimg//1000:06d}', step=cur_nimg)
                 experiment.log_text(text=' '.join(fields_for_comet), step=cur_nimg)
```

Comment:
Why do we need it out the "if" statement?
If experiment is None, is it appropriate for us to call log_text?